crates/bitwarden/src/mobile/crypto.rs

```diff
@@ -51,10 +51,16 @@ pub enum InitUserCryptoMethod {
         /// this.
         pin_protected_user_key: EncString,
     },
+    Passwordless {
+        request_private_key: String,
+        protected_user_key: AsymmetricEncString,
```

Comment:
Could we add some comments on how to get these values? Like we do with PIN and DecryptedKey?